docs/how-to-create-a-new-release.rst

```diff
@@ -103,27 +103,25 @@ still work on a new installation. This is important because it can find missing
 Building the documentation
 --------------------------
 
-An important part of the release process is ensuring that the documentation for readthedocs_ site can be built. This can
-be tested locally by executing the following commands in the repository folder::
+Well documented code is an essential part of the release, allowing your code's legacy to only grow in glory and admiration.
+The documentation will be rendered via the readthedocs_ site, allowing future developers, practitioners, researchers and students
+to understand and build upon your work. Note: entry points created by the command ``cea install-toolbox`` should be run at least once
+and you also need to install GraphViz_ to produce the graphs.
 
-    cd docs
-    make clean
-    make html
+CEA uses sphinx_ to document all module code, and has a tool specific to documentation to make your life a little easier. Simply run via the command
+line once the cea virtual environment is activated in conda: ``activate cea``::
 
-For this to run you will need to first ``pip install sphinx``. Also note that the command ``cea install-toolbox`` needs
-to be run at least once to set up some paths required for importing certain modules. You will also need to do
-``conda install numba``. You also need to install GraphViz_ to produce the graphs.
+``cea-doc html``
 
-If any error messages show up, these need to be fixed before publishing the release. The readthedocs_ site uses
-these steps to produce the developer and API documentation.
+This will remove, rebuild and open any change files, allowing you to check how the documentation htmls were rendered. Please address any
```

Comment:
"remove, rebuild and open any change files" - what is a change file? they get opened?!

Reply:
change file are the files which are recorded as modified by a gitdiff. This way you don't have to go to `docs\build` each time you want to double check how things worked out.